### DISCUSSION TITLE
Possible Bug

### DISCUSSION THREAD

**Discussion by AdamMBC** [ingrammicro/puzzle-publisher#81] [General]
I found a bug that when I:
Enable "Ask custom artboard Size" -> Set a size on export to HTML -> Export file with custom artboards -> Disable "Ask custom Dartboard Size" -> Export file without custom artboards. The custom size does not deactivate. The exported HTML file stays the size of the original custom artboard size.

To fix this, I have to:
Enable "Ask custom Dartboard Size" -> Set it to `0` width and `0` height -> Disable "Ask custom Dartboard Size". The artboard now loads correctly.  

PS: Love the plugin. Very useful!

**Comment by MaxBazarov** [comment]
Thanks for a feedback!
Fixed in the latest 17.8.2 update
